UI/src/view/linear_asset/laneModellingForm.js

```diff
@@ -0,0 +1,597 @@
+      {name: 'date_period', fieldType: self.DatePeriodField},
+      {name: 'hidden_read_only_number', fieldType: self.HiddenReadOnlyFields}
+    ];
+
+  var mainLaneFormStructure = {
+    fields : [
+      {label: 'Kaista', type: 'read_only_number', publicId: "lane_code", weight: 6},
+      {
+        label: 'Kaistan tyypi', required: 'required', type: 'single_choice', publicId: "lane_type", defaultValue: "1", weight: 7,
+        values: [
+          {id: 1, label: 'Pääkaista'}
+          ]
+      },
+      {
+        label: 'Kaista jatkuvuus', required: 'required', type: 'single_choice', publicId: "lane_continuity", defaultValue: "1", weight: 8,
+        values: [
+          {id: 1, label: 'Jatkuva'},
+          {id: 2, label: 'Jatkuu toisella kaistanumerolla'},
+          {id: 3, label: 'Kääntyvä'},
+          {id: 4, label: 'Päättyvä'},
+          {id: 5, label: 'Jatkuva, osoitettu myös oikealle kääntyville'},
+          {id: 6, label: 'Jatkuva, osoitettu myös vasemmalle kääntyville'},
+        ]
+      },
+      {label: 'Kaista ominaisuustieto', type: 'text', publicId: "lane_information", weight: 9}
+    ]
+  };
+
+  var roadAddressFormStructure = {
+    fields : [
+      {label: 'Tie', type: 'read_only_number', publicId: "initial_road_number", weight: 1},
+      {label: 'Osa', type: 'read_only_number', publicId: "initial_road_part_number", weight: 2},
+      {label: 'Etäisyys', type: 'read_only_number', publicId: "initial_distance", weight: 3},
+      {label: 'Osa', required: 'required', type: 'number', publicId: "end_road_part_number", weight: 4},
+      {label: 'Etäisyys', required: 'required', type: 'number', publicId: "end_distance", weight: 5}
+    ]
+  };
+
+    function reloadForm(rootElement){
+      dateutil.removeDatePickersFromDom();
+      rootElement.find('#feature-attributes-header').html(self.renderHeader(self._assetTypeConfiguration.selectedLinearAsset));
+      rootElement.find('#feature-attributes-form').html(self.renderForm(self._assetTypeConfiguration.selectedLinearAsset, false));
+      rootElement.find('#feature-attributes-form').prepend(self.renderPreview(self._assetTypeConfiguration.selectedLinearAsset));
+      rootElement.find('#feature-attributes-form').append(self.renderLaneButtons(self._assetTypeConfiguration.selectedLinearAsset));
+      rootElement.find('#feature-attributes-footer').html(self.renderFooter(self._assetTypeConfiguration.selectedLinearAsset));
+    }
+
+    var AvailableForms = function(){
+      var formFields = {};
+
+      this.addField = function(field, sideCode){
+        if(!formFields['sidecode_'+sideCode])
+          formFields['sidecode_'+sideCode] = [];
+        formFields['sidecode_'+sideCode].push(field);
+      };
+
+      this.getFields = function(sideCode){
+        if(!formFields['sidecode_'+sideCode])
+          Error("The form of the sidecode " + sideCode + " doesn't exist");
+        return formFields['sidecode_'+sideCode];
+      };
```

Comment:
Unused code, check if it can be deleted.

Reply:
Deleted